gstaichi/codegen/cpu/codegen_cpu.cpp

```diff
@@ -262,76 +267,85 @@ void KernelCodeGenCPU::optimize_module(llvm::Module *module) {
     options.NoInfsFPMath = 0;
     options.NoNaNsFPMath = 0;
   }
+
   options.HonorSignDependentRoundingFPMathOption = false;
   options.NoZerosInBSS = false;
   options.GuaranteedTailCallOpt = false;
 
-  llvm::legacy::FunctionPassManager function_pass_manager(module);
```

Comment:
What are the implications of removing these? Not very keen on removing things without a fairly exhaustive descirption please of:
- what they do
- possible consequences of removing them